changelog/issue-7670.md

```diff
@@ -0,0 +1,6 @@
+audience: worker-deployers
+level: minor
+reference: issue 7670
+---
+
+Removed some of the included Azure certificates
```

Comment:
Mind specifying the error we were running into here too please?

Reply:
rewarded changelog again